src/networks.ts

```diff
@@ -0,0 +1,47 @@
+export var networks = {
+    mainnet: {
```

Comment:
[suggestions]: following the structure in `EIP3085` https://eips.ethereum.org/EIPS/eip-3085.
This will be more easier for the external apps to use these configuration.